src/components/office-cards/BaseCard.tsx

```diff
@@ -7,24 +8,26 @@ const BaseCard: React.FC<{
   title: string;
 }> = ({ name, title, imageSrc }) => {
   return (
-    <div className="relative flex items-center bg-blue text-white w-full max-w-3xl rounded-[32.5px] transition-all duration-300 overflow-visible h-[400px]">
+    <div className="relative w-[525px] h-[250px] flex items-center bg-blue text-white max-w-3xl rounded-[32.5px] transition-all duration-300 overflow-visible">
       {/* Officer Image */}
       <div className="absolute left-0 bottom-0 w-1/2 h-auto transform translate-x-[-0.5px] translate-y-[0%] z-0">
         <Image
-          src={imageSrc}
+          src={`/images/samahan-cutouts/${imageSrc}`}
           alt={name}
           width={450}
           height={600}
           className="object-cover"
           priority
         />
       </div>
-
       {/* Text content */}
-      <div className="flex flex-col ml-auto w-1/2 space-y-2 pl-8 pr-8 z-10">
-        <h2 className="font-bold text-3xl leading-tight mt-1">{name}</h2>
-        <p className="text-lg">{title}</p>
-        <div className="absolute bottom-7 right-7 z-20">
+      <div className="flex flex-col justify-between ml-auto w-1/2 space-y-2 p-4 z-10 gap-10">
+        <div className="text-left">
+          <h2 className="font-bold text-thin text-left">{name}</h2>
+          <p className="text-main font-normal">{title}</p>
+        </div>
```

Comment:
To the parent this, I find it unnecessary to do `space-y-2` if you are using gap-10 and flex box.

Also for `ml-auto`, I haven't seen the actual component but this can be improve,
I suggest just do `justify-start` since you're using flex box